tests/annotator/test_count_annotator.py

```diff
@@ -233,5 +233,15 @@ def test_tokenization_edge_cases(self):
                 test_utils.assertHasProps(actual.metadata, expected)
         doc.add_tier(self.annotator)
 
+    # def test_year_count(self):
```

Comment:
No, I commented out the tests because they are failing at the moment, but they will be useful when I'm working on fixes.